e2e/gui/src/test/java/com/epam/pipeline/autotests/DataStoragesCLITest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017-2020 EPAM Systems, Inc. (https://www.epam.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.epam.pipeline.autotests;
+
+import com.epam.pipeline.autotests.ao.ToolTab;
+import com.epam.pipeline.autotests.mixins.Authorization;
+import com.epam.pipeline.autotests.mixins.Navigation;
+import com.epam.pipeline.autotests.utils.C;
+import com.epam.pipeline.autotests.utils.TestCase;
+import com.epam.pipeline.autotests.utils.Utils;
+import com.epam.pipeline.autotests.utils.listener.Cloud;
+import com.epam.pipeline.autotests.utils.listener.CloudProviderOnly;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import static com.epam.pipeline.autotests.ao.Primitive.CLOUD_REGION;
+import static java.lang.String.format;
+
+public class DataStoragesCLITest extends AbstractSinglePipelineRunningTest
```

Comment:
@Listeners(value = ConditionalTestAnalyzer.class) should be added

Reply:
Done